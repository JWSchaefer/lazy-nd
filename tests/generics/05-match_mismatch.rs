use lazy_nd::lazy_nd;
#[lazy_nd(dim = D)]
struct Test<const T: usize, D> {}
fn main() {}
