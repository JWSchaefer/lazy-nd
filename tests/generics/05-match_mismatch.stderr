error: Expected generic `<const D : usize>`
 --> $DIR/05-match_mismatch.rs:3:12
  |
3 | struct Test<const T: usize, D> {}
  |            ^
